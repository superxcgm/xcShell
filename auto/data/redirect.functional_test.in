echo "hello world" > redirect.out
cat redirect.out
echo "hello world" >> redirect.out
cat redirect.out
echo "1 + 1" > bc.input
bc < bc.input
bc < bc.input > bc.output
cat bc.output
echo '1 / 0' | bc 2> a.out
cat a.out
echo '1 / 0' | bc 2>> a.out
cat a.out
echo '1 / 0' | bc > a.out 2>&1
cat a.out
echo '1 / 0' | bc >> a.out 2>&1
cat a.out
echo -e '1 + 1 \n 1 / 0' | bc > a.out 2> b.out
cat a.out
cat b.out