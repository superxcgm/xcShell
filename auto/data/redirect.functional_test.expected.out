xcShell_functional_test > echo "hello world" > redirect.out
xcShell_functional_test > cat redirect.out
hello world
xcShell_functional_test > echo "hello world" >> redirect.out
xcShell_functional_test > cat redirect.out
hello world
hello world
xcShell_functional_test > echo "1 + 1" > bc.input
xcShell_functional_test > bc < bc.input
2
xcShell_functional_test > bc < bc.input > bc.output
xcShell_functional_test > cat bc.output
2
xcShell_functional_test > echo '1 / 0' | bc 2> a.out
xcShell_functional_test > cat a.out
Runtime error (func=(main), adr=3): Divide by zero
xcShell_functional_test > echo '1 / 0' | bc 2>> a.out
xcShell_functional_test > cat a.out
Runtime error (func=(main), adr=3): Divide by zero
Runtime error (func=(main), adr=3): Divide by zero
xcShell_functional_test > echo '1 / 0' | bc > a.out 2>&1
xcShell_functional_test > cat a.out
Runtime error (func=(main), adr=3): Divide by zero
xcShell_functional_test > echo '1 / 0' | bc >> a.out 2>&1
xcShell_functional_test > cat a.out
Runtime error (func=(main), adr=3): Divide by zero
Runtime error (func=(main), adr=3): Divide by zero
xcShell_functional_test > echo '1 + 1' > bc.in
xcShell_functional_test > echo '1 / 0' >> bc.in
xcShell_functional_test > bc < bc.in > a.out 2> b.out
xcShell_functional_test > cat a.out
2
xcShell_functional_test > cat b.out
Runtime error (func=(main), adr=3): Divide by zero
xcShell_functional_test > 